java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -3682,6 +3687,45 @@ public void xreadgroup_return_failures(BaseClient client) {
         }
     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void xack_return_failures(BaseClient client) {
```

Comment:
Should rename since it doesn't return all failures

Reply:
all the xack calls return failures?